express-routes/server.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+
+const app = express();
+
+app.get('/', (req, res) => {
+  res.send('test number one, slash');
+});
+
+app.get('/notes', (req, res) => {
+  res.send('test number two , notes');
+});
+
+app.post('/notes/123', (req, res) => {
+  res.send('test number three, 123');
+});
+
+app.use((req, res, next) => {
```

Comment:
This needs to be first. The middleware routes are processed in order, so if this one is not first, it will never get called.